### PR TITLE
Tickets/PREOPS-1151

### PR DESCRIPTION
**Pull request by kadrlica**
Updates to DP0.2 portal tutorials:
https://jira.lsstcorp.org/browse/PREOPS-1151

### CODE REVIEW

**Review by kadrlica — COMMENTED** [review]
Great updates Melissa! The tutorials look really good!

I've directly committed a few minor changes to the text, and I've added some comments below. Some of these would definitely be slight extensions to the existing tutorials, so feel free to ignore them for now.

(Looks like I can't explicitly approve this PR since I am the original author.)

File: tutorials-examples/portal-beginner.rst [hunk 1]
```diff
@@ -14,99 +14,146 @@
...
 
-.. _DP0-2-Portal-Beginner-Step-4:
+.. figure:: /_static/portal_tut01_step02c.png
+	:name: portal_tut01_step02c
+	
+	The color-magnitude diagram.
 
-Select configure the graph to create a color magnitude diagram
-==============================================================
+**Notice:** The default plot style is a density map, which is appropriate for very large data sets.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-beginner.rst:115]: changed

File: tutorials-examples/portal-beginner.rst [hunk 2]
```diff
@@ -14,99 +14,146 @@
...
+	
+	Set the plot parameters.
 
+2.3. Click "Apply" and then "Close" the pop-up window, and look at the color-magnitude plot.
 
-.. _DP0-2-Portal-Beginner-Step-4:
+.. figure:: /_static/portal_tut01_step02c.png
+	:name: portal_tut01_step02c
+	
+	The color-magnitude diagram.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-beginner.rst:113]: There was a statement to that effect later on but I moved it up and added your exact text.

File: tutorials-examples/portal-beginner.rst [hunk 3]
```diff
@@ -14,99 +14,146 @@
...
 
-After including any other information such as a "Chart title," "X Label," and "Y Label", hit "Apply."
-The plot that you make will be slightly different from what is shown below, and different from the plot in the first of the :ref:`DP0-2-Tutorials-Notebooks` due to random sampling and the 10,000 maximum that we used.
+.. code-block:: SQL
+   SELECT coord_dec,coord_ra,g_calibFlux,i_calibFlux,r_calibFlux 
+   FROM dp02_dc2_catalogs.Object 
+   WHERE detect_isPrimary =1 
+   AND g_calibFlux >360 AND g_extendedness =0 
+   AND i_calibFlux >360 AND i_extendedness =0 
+   AND r_calibFlux >360 AND r_extendedness =0
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-beginner.rst:157]: that's odd I see it rendered just fine -- am resolving convo but we should double check later

File: tutorials-examples/portal-intermediate.rst [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+**Targeted learning level:** intermediate
+
+**Introduction:** 
+This notebook explores the data for a type Ia supernova (SNIa) lightcurve: the i-band photometry, the seeing for each i-band epoch, and the astrometric scatter of the i-band observations.
+The scientific motivation here is that a user knows the coordinates of a low-redshift SNIa (67.4579, -44.0802), and now
+wants to find an i-band epoch in which this SNIa was bright *and* the seeing was good.
+The user also wants to get a sense of the scatter in the astrometric precision for the coordinates in the DiaObjects catalog.
+For example, perhaps the user knows there is pre-explosion JWST imaging of the host galaxy, and is looking for the best image from Rubin Observatory
+with which to co-register with the JWST image in order to characterize the underlying stellar population (or progenitor star) at the SNIa's location.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:35]: Not really possible with great precision in a single case but _statistically_ perhaps something interesting could be derived from a large sample. Usually people get laser AO to get the kind of astrometry needed. (Plus this isn't usually done with SNIa anyway). That's possible for only several SNe per year. So maybe in the future someone will try to do something with a larger sample who knows. I will add some blah blah about that right now.

[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:35]: Went with _"For example, perhaps the user knows there is pre-explosion JWST imaging of the host galaxy, and is looking for the best image from Rubin Observatory with which to co-register with the JWST image in order to characterize the underlying stellar population at the SNIa's location. (Note that typically, laser adaptive optics is necessary to obtain ground-based images of SNe that have high enough astrometric precision to co-register with a space-based image and identify a progenitor star, so this use-case is a bit contrived)."_

File: tutorials-examples/portal-intermediate.rst [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+
+.. _DP0-2-Portal-Intermediate_Step-1:
+
+1. Determine the DiaObject Id
+=============================
+
+1.1. Log in to the Portal Aspect.
+
+1.2. Leave the TAP Service and Query Type as their default, and select the dp02_dc2_catalogs.DiaObject table.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:51]: Made dp02_dc2_catalogs.DiaObject  a link to schema

File: tutorials-examples/portal-intermediate.rst [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    The r-band lightcurve for the SNIa of interest.
+
+3.3. To add a plot of time versus seeing: use the settings icon, choose "Add New Chart" and match the parameters shown below, then click "OK".
+
+.. figure:: /_static/portal_tut02_step03c.png
+    :name: portal_tut02_step03c
+
+    Plot parameters for the time *versus* seeing plot.
+
+3.4. To add a plot to visualize the astrometric precision: use the settings icon, choose "Add New Chart" and match the parameters shown below, then click "OK".
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:120]: Updated in text and screenshot.

File: tutorials-examples/portal-intermediate.rst [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+Note that in both the X and Y parameters, the difference between the DiaSource coordinate and the DiaObject coordinate are divided by 3600, so that the plot axes are in arcseconds.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:121]: yes, super, added

File: tutorials-examples/portal-intermediate.rst [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+In the center plot, click on the r-band epoch with the best seeing (0.75 arcsec).
+Notice how the point turns orange in all three plots, and that the corresponding table row will be highlighted.
+
+In the left-most plot, the lightcurve, notice that for this "best-seeing" epoch the SNIa had an apparent magnitude near its peak (around 22nd mag).
+That makes it a suitable choice for the scientific use-case outlined in the Introduction.
+
+In the right-most plot, the astrometric precision, notice that for this "bright / best-seeing" epoch the measured sky coordinates of the DiaSource are very close to those reported for the DiaObject.
+This *does not* necessarly mean that the coordinates for the "best-seeing" epoch are more accurate, because the
+coordinates of DiaObjects are *derived from* the individual DiaSources.
+The point of the right-most plot is more that the overall scatter is low (less than 0.3 arcsec), and that selecting the
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-intermediate.rst:138]: Just say "the overall scatter is less than 0.3 arcsec". No (or little) bias from host galaxy in astrometry from difference images.

File: tutorials-examples/portal-images.rst [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+2.4. Mark and center the SNIa.
+Choose the "center" icon (hover-over text "Image center drop down"), and in the box next to "Center On" enter the SNIa's coordinates, "67.4579, -44.0802", and then click "Go & Mark".
+If you pan away from the SNIa, recenter using the "center" icon and finding the SNIa coordinates under "Recent Positions".
+
+.. figure:: /_static/portal_tut03_step02c.png
+    :name: portal_tut03_step02c
+
+    Center on and mark the coordinates of the SNIa.
+
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-images.rst:90]: added

File: tutorials-examples/portal-images.rst [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+.. _DP0-2-Portal-Images_Step-2:
+
+2. Interact with the deepCoadds
+===============================
+
+2.1. In the results view, click on "img-tbl" in the upper right corner to remove the "xy plot" from the display as it is not needed.
+
+2.2. Above the image, click on the grid icon (hover-over text "Show full grid") to simultaneously view all six filters' deepCoadds. The default view is to center all deepCoadds on the center of the patch.
+In this example the compass has been enabled using the "Tools" icon (wrench and ruler, hover-over text "Tools drop down").
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply tutorials-examples/portal-images.rst:66]: yes, made sure at least one screen shot had the filter column, meant to add more. what i really want is a way to label images based on column content but couldn't figure out yet.

added text about grid order matching table order and filtering on em_min

File: tutorials-examples/portal-beginner.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (kadrlica) [reply tutorials-examples/portal-beginner.rst:159]: Oh, sorry. I committed a change to add the necessary blank space after defining the code block, so it should render correctly now (and it sounds like it does).

**Pull request merged**